From 3d06b038ecc45bc5bf80399aa6574b7b828d61b6 Mon Sep 17 00:00:00 2001
From: "G.h. Gao" <guanhua.gao@nxp.com>
Date: Fri, 14 Jun 2019 15:14:37 +0800
Subject: [PATCH 539/741] staging: fsl-dpaa2: Remove duplicate config options

commit 9be87422d1c55985db028294b5fcf2ddc4f2a7e0 from
http://source.codeaurora.org/external/qoriq/qoriq-components/linux

Remove dpaa2-eth config options left over from when the
driver was located in staging.

Signed-off-by: Ioana Radulescu <ruxandra.radulescu@nxp.com>
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 drivers/staging/fsl-dpaa2/Kconfig | 37 -------------------------------
 1 file changed, 37 deletions(-)

diff --git a/drivers/staging/fsl-dpaa2/Kconfig b/drivers/staging/fsl-dpaa2/Kconfig
index 28cd595e97a2..f5bf2f3fb6db 100644
--- a/drivers/staging/fsl-dpaa2/Kconfig
+++ b/drivers/staging/fsl-dpaa2/Kconfig
@@ -18,42 +18,5 @@ config FSL_DPAA2_ETHSW
 	Driver for Freescale DPAA2 Ethernet Switch. Select
 	BRIDGE to have support for bridge tools.
 
-if FSL_DPAA2_ETH
-config FSL_DPAA2_ETH_USE_ERR_QUEUE
-	bool "Enable Rx error queue"
-	default n
-	---help---
-	  Allow Rx error frames to be enqueued on an error queue
-	  and processed by the driver (by default they are dropped
-	  in hardware).
-	  This may impact performance, recommended for debugging
-	  purposes only.
-
-# QBMAN_DEBUG requires some additional DPIO APIs
-config FSL_DPAA2_ETH_DEBUGFS
-	depends on DEBUG_FS
-	bool "Enable debugfs support"
-	default y
-	---help---
-	  Enable advanced statistics through debugfs interface.
-
-config FSL_DPAA2_ETH_DCB
-        bool "Data Center Bridging (DCB) Support"
-        default n
-        depends on DCB
-        ---help---
-          Say Y here if you want to use Data Center Bridging (DCB) features
-          (PFC) in the driver.
-
-          If unsure, say N.
-
-config FSL_DPAA2_ETH_CEETM
-	depends on NET_SCHED
-	bool "DPAA2 Ethernet CEETM QoS"
-	default n
-	---help---
-	  Enable QoS offloading support through the CEETM hardware block.
-endif
-
 source "drivers/staging/fsl-dpaa2/mac/Kconfig"
 source "drivers/staging/fsl-dpaa2/evb/Kconfig"
-- 
2.17.1

